### ISSUE TITLE
GET /v2/registrations

### ISSUE THREAD

**Issue by jmcanterafonseca** [telefonicaid/fiware-orion#3005]
implement GET /v2/registrations so that context source registrations can be listed. 

For the time being, querying registrations is out of scope but it will be a feature to be taken into account.

**Comment by fgalan** [comment]
Have a look to general comment on the registraitons functionality at https://github.com/telefonicaid/fiware-orion/issues/3008. Many of them could also apply to this issue.

**Comment by fgalan** [comment]
All the registration features share the same common base branch: *feature/ngsiv2_registrations*.

Thus, this issue should be implemented in a branch splited from feature/ngsiv2_regsitrations (instead of master), using the task/ prefix in the new branch name, and the PR done using feature/ngsiv2_registrations as base branch (instead of master).

feature/ngsiv2_registrations will be update regularly with master, to keep it as synced as possible and ease the final feature/ngsiv2_registrations -> master merge once all the NGSIv2 registrations management  functionality gets completed.

**Comment by fgalan** [comment]
The "core" of the functionality (except pagination) implemented in https://github.com/telefonicaid/fiware-orion/pull/3101

**Comment by fgalan** [comment]
Pagination implemented in PR https://github.com/telefonicaid/fiware-orion/pull/3105

**Comment by fgalan** [comment]
The basic functionality is completed. Advance functionality has been moved to spin off issues (see https://github.com/telefonicaid/fiware-orion/issues/3008#issuecomment-365253516 for the details of those spin off issues).

The issues is thus closeable once **feature/ngsiv2_registrations lands on master**. Assigned to milestone 1.12.0, as this is probably the version in which this landing will take place.
